creator-node/src/diskManager.ts

```diff
@@ -537,24 +537,48 @@ async function _copyLegacyFiles(
   return copiedPaths
 }
 
-async function _migrateNonDirFilesWithLegacyStoragePaths(
+/**
+ * Creates pagination windows and calls the given function to query for CIDs within the windows.
+ * The [min, max] windows are:
+ * - [Qmaaa, Qmaaz] to [Qmzza, Qmzzz]; and
+ * - [Qmaa0, Qmaa9] to [Qmz90, Qmz99]
+ *
+ * We don't have any CIDs that start with Qm[0,9], so that's why we:
+ * - start at Qma00 instead of Qm000; and
+ * - end at Qmz99 instead of Qm999
+ * @param func the function to call on each pagination window (there are 1872 windows)
+ */
+async function _callFuncOnAllCidsPaginated(
+  func: (minCid: string, maxCid: string) => Promise<void>
+) {
+  // We don't have any CIDs that start with Qm[0,9], so just Qm[a,z] is sufficient
+  for (const char1 of getCharsInRanges('az')) {
+    // Loop [Qm*a,Qm*z] and [Qm*0,Qm*9]
+    for (const char2 of getCharsInRanges('az', '09')) {
+      // For each permutation, search in the pagination window of a-z and 0-0.
+      // I.e., window Qm**a through Qm**z and window Qm**0 through Qm**9
+      await func('Qm' + char1 + char2 + 'a', 'Qm' + char1 + char2 + 'z')
+      await func('Qm' + char1 + char2 + '0', 'Qm' + char1 + char2 + '9')
```

Comment:
hahahaha god this code is hilarious
but hey if it works it works, lets just ship and rip